src/mkdocstrings/templates/python/material/class.html

```diff
@@ -22,7 +22,14 @@
           class="doc doc-heading",
           toc_label=class.name) %}
 
-        <code>{% if show_full_path %}{{ class.path }}{% else %}{{ class.name }}{% endif %}</code>
+        <code>
+          {% if show_full_path %}{{ class.path }}{% else %}{{ class.name }}{% endif %}
+          {% if config.show_bases %}
+            ({% for base in class.bases -%}
+              {{ base|brief_xref() }}{% if base != class.bases[-1] %}, {% endif %}
```

Comment:
I believe the intent here is to skip for the comma after the last item? Then you can use `{% if not loop.last %}` instead. See https://jinja2docs.readthedocs.io/en/stable/templates.html#for

Reply:
Yeah, thanks.  I hadn't used jinja before this.